operator/src/test/java/oracle/kubernetes/operator/DomainProcessorTestSetup.java

```diff
@@ -34,7 +34,9 @@ public static void defineRequiredResources(KubernetesTestSupport testSupport) {
   }
 
   private static V1Secret createSecret() {
-    return new V1Secret().metadata(new V1ObjectMeta().name(SECRET_NAME).namespace(NS));
+    return new V1Secret().metadata(new V1ObjectMeta().name(SECRET_NAME).namespace(NS))
```

Comment:
The varied indentation of the keys in the data map makes this harder to read. Either put them on the same line or line them up vertically.

Reply:
put on the same line.